pymysqlreplication/tests/test_basic.py

```diff
@@ -1486,7 +1486,7 @@ def test_gtid_event(self):
         self.stream = BinLogStreamReader(
             self.database,
             server_id=1024,
-            only_events=[GtidEvent],
+            only_events=[GtidEvent, FORMAT_DESCRIPTION_EVENT],
```

Comment:
```suggestion
            only_events=[GtidEvent, FormatDescriptionEvent],
```
sorry we should change like this.

Reply:
@sean-k1 Applied!